--- 
authors: Freek Pols
updated: december 3, 2025    
---
# Bepaling van soortelijke warmte van water


# Introductie

Water heeft een enorme buffercapaciteit: je moet veel energie toevoegen om de temperatuur van water een graad te verwarmen. In dit practicum gaan we de soortelijke warmte van water bepalen door een bekende hoeveelheid water te verwarmen met een bekende hoeveelheid energie, en de temperatuurstijging te meten.

# Theorie

Geef hier de natuurkundige achtergrond van het onderwerp. 

# Methode en materialen

## Ontwerp
Een waterbad met bekende massa aan water wordt verwarmd met een elektrisch verwarmingselement dat een bekende hoeveelheid energie levert.
De temperatuur van het water wordt gemeten met een temperatuursensor.
Door de temperatuurstijging als functie van de tijd te meten kan de soortelijke warmte van water worden berekend.


## Materialen
Hieronder staat de lijst van benodigde materialen bij deze proef:
- Maatbeker
- Weegschaal
- Water
- Elektrisch verwarmingselement ($10 \mathrm{\Omega}$, $10 \mathrm{W}$)
- Voedingsbron 
- Thermometer of temperatuursensor
- Stopwatch of timer

```{figure} figures/cwater_setup.jpg
:width: 70%

Een schematische weergave van de opstelling
```

## Procedure
```{exercise}
1. Beschrijf de procedure die nodig is om de soortelijke warmte van water te bepalen.
2. Geef jouw beschrijving van de procedure aan een ander team, zij verzamelen voor jou de data op basis van de gegevens.
3. Voer het experiment van een ander team uit op basis van hun beschrijving.
4. Analyseer de data die het andere groepje voor jou heeft verzameld.
```


## Veiligheid
We maken gebruik van een $10 \mathrm{\Omega}$, $10 \mathrm{W}$ weerstand.
Deze wordt snel heet.
De bronspanning mag dan ook alleen aan wanneer de weerstand in het water zit.
Raak de weerstand niet aan tijdens het experiment.
Omdat de weerstand in het water zit, kunnen we wel het elektrisch vermogen hoger zetten zonder dat de weerstand oververhit raakt.
Het maximaal vermogen mag $40 \mathrm{W}$ zijn.
Daarbij moet de roerder wel aanstaan om de warmte goed te verdelen.

## Data analyse
De soortelijke warmte kan bepaald worden met de formule c = Q/ mΔT. Verschil in temperatuur is gemeten door een thermometer. Vervolgens kan de warmte overdracht worden bepaald door de formule U keer I keer t, waarbij dit de spanning en stroomsterkte is van het verwarmings element. Met deze gegevens kan de soortelijke warmte worden bepaald. 

# Resultaten


In [1]:
T_b = 19.1
T_e = 20.1
U = 11.8
I = 1.21
V = 420e-3
m_w = V * 997e-3
t = 105 
delta_T = T_e - T_b 
Q = U*I * t
def soortelijke_warmte(m, Q):
    return Q / (m * delta_T)
print('De soortelijke warmte van water is %.0f J/KgK' %(soortelijke_warmte(m_w, Q)))
print("Kunnen geen grafiek maken van de hoeveelheid water tegen over de het verschil temperatuur door hoeveelheid datapunten")

De soortelijke warmte van water is 3580 J/KgK
Kunnen geen grafiek maken van de hoeveelheid water tegen over de het verschil temperatuur door hoeveelheid datapunten


# Discussie en conclusie

De soortelijke warmte is berekend door de tijd en temperatuur verschil te meten tussen twee verschillende situaties. Uit deze berekeningen is gekomen dat de soortelijkewarmte gelijk is aan 3580 J/KgK. Bij het experiment zijn maar twee datapunten gebruikt om de soortelijke warmte te berekene, echter kan de soortelijke warmte nauwkeuriger worden bepaald als de temperatuur op meerdere momenten was gemeten. Vervolgens kan aan de hand van een curve fit de soortelijke warmte worden bepaald. Hierdoor had de onzekerheid van de gemeten waarde kleiner geworden. De gevonden waarde is waarnschijnlijk een redelijke indicatie van de soortelijke warmte van water, maar is niet nauwkeurig. 